### ISSUE TITLE
Too many dependencies

### ISSUE THREAD

**Issue by paulvi** [invesdwin/invesdwin-context-python#4]
I understand that this is maybe non addressable issue,
but this project has too many dependencies.

I hope in future it would be possible to use Python engine modules without all that can be used in 
`invesdwin-context-integration` and `invesdwin-context`

![image](https://user-images.githubusercontent.com/1614482/121346524-cedfa380-c958-11eb-85a5-c0c378a52748.png)

**Comment by subes** [comment]
Yes, it should be possible to make this more concise (similar to https://github.com/invesdwin/invesdwin-nowicket) and not depend on invesdwin-context. Definitely something that could/should be improved at some point.

For now you can only disable unwanted features at runtime: https://github.com/invesdwin/invesdwin-context-python#avoiding-bootstrap
But you can not remove the dependencies yet. Some work needs to be done to change the way in which the properties get loaded for this and the spring-beans need to be separated from the actual singletons. Then also the coupling into invesdwin-context needs to separated into its own module. Requires some work, but is definitely doable if required (e.g. to use this in some restricted environment). For cases where truncating jars is not good enough.
